src/content/docs/tutorial-service-level-mgmt/establish-capability-sli.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Establish capability SLIs"
+metaDescription: "New Relic observability maturity series: our service level management guide shows you how to easily measure and communicate the overall health, performance, and quality of your digital products and services to all stakeholders."
+redirects:
+freshnessValidatedDate: never
+---
+
+import queriesnrqlTransactionListNrql from 'images/queries-nrql_screenshot-full_transaction-list-nrql.webp'
+
+import apmCapabilityFilterClause from 'images/apm_screenshot-crop_capability-filter-clause.webp'
+
+Having service levels on critical business capabilities, like login or authorize payment, will rapidly close the language barrier between IT and business. Service level scores on capabilities also give you better opportunities to fix service levels when they begin to degrade. For example, if the login service level begins to degrade, you'll know to look at identity mangement dependencies and workflows starting at the consumer-facing API.
+
+You don't need to have intimate knowledge of an application or service in order to complete this task. You only need to find the consumer-facing API (service boundary) and follow the steps below.
+
+1. Identify the service boundary application as described in the [establishing an output SLI](/docs/tutorial-service-level-mgmt/establish-output-sli/) doc.
+2. Run the following NRQL query to identify baselines on the most frequently used transactions. Remember to replace `Order-Processing` with the application name you identified.
+
+```
+FROM Transaction SELECT count(*), percentile(duration, 95) WHERE appName='Order-Processing' FACET name SINCE 1 WEEK AGO
+```
+
+You should see something similar to the screenshot below.
+
+<img
+  alt="transaction list query"
+  src={queriesnrqlTransactionListNrql}
+/>
+
+You'll see the first transaction states it has something to do with "purchase." You can now create a "purchase" capability service level.
```

Comment:
this seems very critical and like the "goal" for steps 1 and 2.

Reply:
I've moved stuff around and added the goal to the intro sentence